pkg/networkutils/network.go

```diff
@@ -70,6 +70,8 @@ const (
 	// - kube-proxy uses 0x0000c000
 	// - Calico uses 0xffff0000.
 	defaultConnmark = 0x80
+
+	ethernetMTU = 9001
```

Comment:
Can you add a comment here to indicate that 9001 also need to be defined in driver.go?

Reply:
Updated to add comments where MTU is defined.